src/domain/event/eventContent/__tests__/EventContent.test.tsx

```diff
@@ -88,3 +87,16 @@ it('should render event content fields', async () => {
     expect(screen.queryAllByRole(role, { name })).toHaveLength(2);
   });
 });
+
```

Comment:
we could also have another counterpart test that checks that map _is_ present when id is something else